### ISSUE TITLE
how to use

### ISSUE THREAD

**Issue by daomengRen** [Zoha/movier#194]
sorry, I'm a new js coder. can you provide some examples? thanks
Error：
```
node_modules/movier/dist/src/titleDetailGetters.js:24
        throw new Error("failed to get the result from IMDB : " + e.message);
              ^

Error: failed to get the result from IMDB : Request failed with status code 403
    at getTitleDetailsByUrl (node_modules/movier/dist/src/titleDetailGetters.js:24:15)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)
```

**Comment by daomengRen** [comment]
yeah,looking forward to the new version, thank you

**Comment by Zoha** [comment]
@daomengRen fixed in version 2.3.0
please upgrade
